package.json

```diff
@@ -17,14 +17,16 @@
     "watch:angular1": "grunt delta",
     "deploy:build2api": "ng build --deleteOutputPath=true --optimization=true --configuration production --output-path dist",
     "deploy": "run-s -l build:angular1 deploy:build2api",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test": "ng test",
+    "test:ci": "ng test --karma-config=src/karma-ci.conf.js --no-progress"
   },
   "keywords": [],
   "author": "",
   "dependencies": {
     "@angular/animations": "^13.1.1",
     "@angular/cdk": "~13.1.1",
-    "@angular/cli": "~13.1.2",
+    "@angular/cli": "~13.1.1",
```

Comment:
We have to use consistent versions across angular packages (ie. `13.1.1`) to avoid surprise compatibility issues.